packages/apps/plugins/plugin-markdown/src/components/DocumentEditor.tsx

```diff
@@ -36,9 +64,10 @@ const DocumentEditor = ({ document: doc, extensions: _extensions = [], ...props
           setFallbackName(doc, text);
         },
       }),
-      _extensions,
+      providerExtensions,
+      baseExtensions,
     ],
-    [doc, doc.content, _extensions, identity],
+    [doc, doc.content, baseExtensions, identity],
```

Comment:
should include provide extensions too?

Reply:
also space if it's being computed outside the memo

Reply:
https://www.npmjs.com/package/eslint-plugin-react-hooks 👀